modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,20 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+The amount of resources required by Network Observability depends on the size of your cluster and your requirements for ingesting and storing observability data. There are settings to consider that can help you get an optimal setup and observability.
+
+The following settings can help you manage resources and performance from the outset:
+   
+    eBPF Sampling:: You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Lower sampling values might consume a large amount of computational, memory and storage resources. You can mitigate this by specifying a sampling ratio value. A value of 100 means 1 flow every 100 is sampled. A value of 0 or 1 means all flows are captured. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, so 1 flow every 50 is sampled. Note that more sampled flows also means more storage needed. It is recommend to start with default values and refine empirically, to determine which setting your cluster can manage.
+
+    Restricting or excluding interfaces::  Reduce the overall observed traffic using `spec.agent.ebpf.interfaces` and `spec.agent.ebpf.excludeInterfaces`. By default, the agent fetches all the interfaces in the system, except the ones listed in `excludeInterfaces` and 'lo' (local interface). Note that the interface names might vary according to the CNI used.
```

Comment:
```suggestion
    Restricting or excluding interfaces::  Reduce the overall observed traffic by setting values for `spec.agent.ebpf.interfaces` and `spec.agent.ebpf.excludeInterfaces` . By default, the agent fetches all the interfaces in the system, except the interfaces listed in `excludeInterfaces` and 'lo' (local interface). Note that the interface names might vary according to the CNI used.
``

Reply:
Spell out CNI? I'm not familiar with this acronym but maybe it's common knowledge in this specialized area?

Reply:
>`spec.agent.ebpf.excludeInterfaces` .
Not sure why you need the space here. 

Should spell out CNI.

Reply:
Addressed